app/assets/stylesheets/nucore-overrides.scss

```diff
@@ -9,3 +9,21 @@ Example of this to include here:
   $headerBackgroundColor: #571963;
   $headerColor: white;
 */
+
+// Shared
+$darkGrey: #3F3B26;
+$grey: #9B9B9B;
+$lightGrey: #D4D1D1;
+$lighterGrey: #F3F3F3;
+
+// Manage Mode
+$orange: #FF7126;
+$orange: #F89662;
```

Comment:
Looks like you're defining `$orange` twice. Maybe one is supposed to be `$darkOrange`?